src/LanguageServer/Impl/Implementation/Server.Completion.cs

```diff
@@ -38,7 +40,7 @@ public sealed partial class Server {
             var analysis = entry != null ? await entry.GetAnalysisAsync(50, cancellationToken) : null;
             if (analysis == null) {
                 TraceMessage($"No analysis found for {uri}");
-                return new CompletionList();
+                return EmptyCompletion;
```

Comment:
It is not safe to return shared instance cause `CompletionList` isn't immutable,

Reply:
OK

Reply:
We do return hover though

Reply:
Considering amount of objects created by Json.Net, another instance won't make a big difference.